pyasic/config/pools.py

```diff
@@ -396,6 +416,25 @@ def as_auradine(self, user_suffix: str = None) -> dict:
             }
         return {"updatepools": {"pools": PoolGroup().as_auradine()}}
 
+    def as_epic(self, user_suffix: str = None) -> dict:
+        if len(self.groups) > 0:
+            return {
+                "pools": {
+                    "coin": "Btc",
+                    "stratum_configs": [
+                        g.as_epic(user_suffix=user_suffix) for g in self.groups
+                    ],
+                    "unique_id": True,
```

Comment:
Since you are prefixing your own suffix, you may want to have unique_id default to False. Most likely would make it more consistent with other FW